inscriptions/ISic0256.xml

```diff
@@ -178,28 +183,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR127510" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="VM">
                 <ab>
-                    <lb n="0"/>
-                    <gap reason="lost" extent="unknown" unit="line"/>
-                    <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">si</supplied>BÍ
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="0"/>
-                    <gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="0"/><gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">si</supplied>BÍ<gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
the Í should presumably be encoded, using `<hi rend=" ">` (whether "tall" or otherwise - if the photo shows what it actually has on it that the accent is here representing)?

Reply:
I think it's one of those cases in which I can't find the photo... I will check. But, yes, in the other cases I used the <hi> element

Reply:
*couldn't

Reply:
`<hi rend="tall">`

Reply:
I'd go with that, since that seems to be what the  Í is normally representing in this edition